readthedocs/projects/tasks.py

```diff
@@ -238,9 +239,18 @@ def run(self, version_pk):  # pylint: disable=arguments-differ
         try:
             self.version = self.get_version(version_pk)
             self.project = self.version.project
-            before_vcs.send(sender=self.version)
+
+            environment = LocalBuildEnvironment(
```

Comment:
Don't we want to not have `LocalBuildEnvironment`  always, but respect the Docker settings?

Reply:
Yes, but not in this PR (this code is 100% compatible with the current implementation of .com). This PR does the change to respect the docker setting https://github.com/readthedocs/readthedocs.org/pull/6436